src/bundle/Resources/public/js/scripts/fieldType/ezdatetime.js

```diff
@@ -65,19 +65,21 @@
         time_24hr: true,
         formatDate: (date) => new Date(date).toLocaleString(),
     };
-    const updateInputValue = (sourceInput, date) => {
+    const updateInputValue = (sourceInput, dates) => {
         const event = new CustomEvent(EVENT_VALUE_CHANGED);
 
-        if (!date.length) {
+        if (!dates.length) {
             sourceInput.value = '';
             sourceInput.dispatchEvent(event);
 
             return;
         }
 
-        date = new Date(date[0]);
-        sourceInput.value = Math.floor(date.getTime() / 1000);
+        const selectedDate = moment(dates[0]);
+        const selectedDateWithUserTimezone = selectedDate.tz(eZ.adminUiConfig.timezone, true);
```

Comment:
What `true` parameter does?
Should use the timezone helper.

Reply:
I didn't use timezone helper, because I wanted to pass `true`, with `true` _tz_ changes timezone without changing hours, minutes, seconds. We need different timezone to correctly calculate timestamp with `valueOf`.

Reply:
So the helper should be enhanced to accept the `true` parameter (default: false).

Reply:
Enhance it here? Base branch is 1.4.

Reply:
Yes, why not.